content/en/docs/reference/glossary/graceful-eviction.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Graceful eviction
+id: graceful-eviction
+date: 2021-04-27
+full_link: /docs/concepts/scheduling-eviction/pod-eviction/
+short_description: >
+  Graceful eviction is the process by which you create an `Eviction` object using
```

Comment:
I'm not sure that backticks (and other Markdown formatting) in a short description will work how you might expect. I'd remove them unless we're confident they always render OK.